ruby/spec/cucumber/ci_environment/remove_user_info_from_url_spec.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+require 'cucumber/ci_environment'
+
+describe Cucumber::CiEnvironment, '#remove_user_info_from_url' do
```

Comment:
- The URI API has a method called `userinfo`. The single-word spelling is intentional. This rename is probably not wanted. That is the file name, and the method name as mentioned in this `describe`

Reply:
Ah I didn't see that, will revert the name